### PR TITLE
Add dataset Availability to dataset profile. 

### PR DESCRIPTION
**Pull request by nicoschif**
Adding the datasetAvailability property to the dataset profile

### CODE REVIEW

File: model/Dataset/Vocabulary/DatasetAvailability.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DatasetAvailability
```
Comment (armintaenzertng) [inline model/Dataset/Vocabulary/DatasetAvailability.md:3]: The name of the vocabulary is the same as that of the property, which will likely lead to confusion. It seems to be common to append "Type" at the end of the vocabulary in that case, i.e. `DatasetAvailabilityType`.

File: model/Dataset/Classes/Dataset.md [hunk 2]
```diff
@@ -58,3 +58,7 @@ Metadata information that can be added to a a dataset that maybe used in a softw
   - type: xsd:string
   - minCount: 0
   - maxCount: 1
+- datasetAvailability
+  - type: xsd:string
```
Comment (armintaenzertng) [inline model/Dataset/Classes/Dataset.md:62]: I presume it is planned to use the newly introduced vocabulary here, not just a string?

File: model/Dataset/Properties/datasetAvailability.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Description
+
+Some datasets are publicly available and can be downloaded directly. Others are only accessible behind a clickthrough, or after filling a registration form. This field will describe the dataset availability from that perspective.
+
+## Metadata
+
+- name: datasetAvailability
+- Nature: DataProperty
+- Range: xsd:string
```
Comment (armintaenzertng) [inline model/Dataset/Properties/datasetAvailability.md:17]: see above

File: model/Dataset/Vocabulary/DatasetAvailability.md (hunk 1 above)
Comment (bennetkl) [reply model/Dataset/Vocabulary/DatasetAvailability.md:3]: Also think renaming with append of "Type" will make it clearer.  Good idea

File: model/Dataset/Classes/Dataset.md (hunk 2 above)
Comment (bennetkl) [reply model/Dataset/Classes/Dataset.md:62]: Yes, it's to use the new definitions.  What did we not do correctly?   Nic listed all the possible 'strings' that would be accepted for this field.   @nicoschif with  the new concept of 'models pretrained with data, should we add an additional  term to the list?

Comment (armintaenzertng) [reply model/Dataset/Classes/Dataset.md:62]: The field should then state 
`- type: datasetAvailabilityType`
(provided the vocabulary is renamed to end with "Type")

**Review by kestewart — APPROVED** [review]
Thanks,  looks good to me for including into dataset profile, and getting it merged.

If there are further tweaks, we can do it in main directly.

File: model/Dataset/Vocabulary/DatasetAvailability.md (hunk 1 above)
Comment (kestewart) [reply model/Dataset/Vocabulary/DatasetAvailability.md]: New version with Type added.

**Pull request merged**